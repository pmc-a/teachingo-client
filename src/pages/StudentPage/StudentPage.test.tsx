import React from 'react';
import { shallow } from 'enzyme';

import Component from './StudentPage';

describe('StudentPage', () => {
    it('should successfully render', () => {
        expect(shallow(<Component />)).toMatchSnapshot();
    });
});
