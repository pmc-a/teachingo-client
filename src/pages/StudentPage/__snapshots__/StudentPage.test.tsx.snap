// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StudentPage should successfully render 1`] = `
<div>
  This is the student view yo!
</div>
`;
